### DISCUSSION TITLE
New release for NonGNU ELPA / MELPA Stable

### DISCUSSION THREAD

**Discussion by denisw** [magit/magit#5187] [Ideas]
I am looking into moving off MELPA to more stable sources like MELPA Stable, but noticed that the last Magit release (v3.3.0) was tagged back in 2021, meaning that I would lose 3 years of awesome new features and bugfixes. The same problem applies to NonGNU ELPA.

Would it be possible to create a v3.4.0 (v4.0.0?) release to fix this issue? I know that it might not be as essy as just creating a Git tag - writing an overview of what‘s new is probably a huge effort at this point - but the effort will probably only grow as the diff from v3.3.0 to master becomes larger and larger. 😅

**Comment by tarsius** [comment]
Less than a week now.
